Launcher.py

```diff
@@ -118,46 +118,28 @@ def handle_uri(path: str, launch_args: Tuple[str, ...]) -> None:
     launch_args = (path, *launch_args)
     client_component = []
     text_client_component = None
-    if "game" in queries:
-        game = queries["game"][0]
-    else:  # TODO around 0.6.0 - this is for pre this change webhost uri's
-        game = "Archipelago"
+    game = queries["game"][0]
```

Comment:
Personally, I'd rather this stay forever. 
The official webhost is not the only thing that can generate `archipelago://` urls, and making game a mandatory parameter seems unhelpful.

Reply:
I'm fine with pushing this down the road if need be but I greatly disagree with keeping it perpetually when forked webhosts can just update, and adding queries to some custom version is not difficult.

Reply:
Honestly, my issue is less with webhost forks, and more for other software entirely.
If I have a "Open Text Client" button in my game client, I'd rather not have to do a `&game=Archipelago` (or worse `&game=` to make the URL work.